website/cue/reference/components/sinks/gcp_stackdriver_logs.cue

```diff
@@ -30,12 +30,8 @@ components: sinks: gcp_stackdriver_logs: {
 			proxy: enabled: true
 			request: {
 				enabled:                    true
-				concurrency:                5
 				rate_limit_duration_secs:   1
```

Comment:
I think you can remove this one as well.

Reply:
Right you are. Resolved in e5c199b8a